dev/conbench_envs/benchmarks.env

```diff
@@ -48,3 +48,4 @@ PARQUET_BUILD_EXAMPLES=ON
 PARQUET_BUILD_EXECUTABLES=ON
 PYTHON=python
 LD_LIBRARY_PATH=$CONDA_PREFIX/lib:$LD_LIBRARY_PATH
+JAVA_OPTIONS=-Xmx8G
```

Comment:
@danepitkin just curious, should we set `Xms` as well?

Reply:
Btw, would it be possible to make this CLI args? Just curious.

Reply:
Thanks for trying this out, @danepitkin 🙂 If it turns out that tweaking the environment configuration fixes the error, I think leaving that configuration here (in the `dev/conbench_envs` location, which is only used by the Ursa machine) is the best choice. 

Reason is we support running the `voltrondata-labs/benchmarks` in other locations too, like local development, so I wouldn't want to always clobber the memory options of users' machines with a CLI arg there.

Reply:
Makes sense to me! Isolating the fix to the smallest scope is the preferred option IMO.